src/app/shared/models/admin-import-export.model.ts

```diff
@@ -0,0 +1,55 @@
+export interface EmailsEdrpous {
+  edrpous: Object;
+  emails: Object;
+}
+
+export interface EmailsEdrpousResponse {
+  edrpous: number[];
+  emails: number[];
+}
+export interface ValidProviders {
```

Comment:
This interface represents a single provider, so do not use the plural for naming, same for Providers and ProvidersID (Id should be in camel case)